stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/internal/ReplyMessageInnerEntity.kt

```diff
@@ -0,0 +1,95 @@
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.offline.repository.domain.message.internal
+
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import androidx.room.Relation
+import io.getstream.chat.android.models.MessageSyncType
+import io.getstream.chat.android.models.SyncStatus
+import io.getstream.chat.android.offline.repository.domain.message.attachment.internal.AttachmentEntity
+import java.util.Date
+
+internal data class ReplyMessageEntity(
+    @Embedded val replyMessageInnerEntity: ReplyMessageInnerEntity,
+    @Relation(entity = AttachmentEntity::class, parentColumn = "id", entityColumn = "messageId")
+    val attachments: List<AttachmentEntity>,
+)
+
+@Entity(tableName = REPLY_MESSAGE_ENTITY_TABLE_NAME)
+internal data class ReplyMessageInnerEntity(
+    @PrimaryKey
+    val id: String,
+    val cid: String,
+    val userId: String,
+    /** the message text */
+    val text: String = "",
+    /** the message text formatted as html **/
+    val html: String = "",
+    /** message type can be system, regular or ephemeral */
+    val type: String = "",
+    /** if the message has been synced to the servers, default is synced */
+    val syncStatus: SyncStatus = SyncStatus.COMPLETED,
+
+    val syncType: MessageSyncType? = null,
+
+    // val syncContent: MessageSyncContentEntity? = null,
+
+    /** the number of replies */
+    val replyCount: Int = 0,
+    /** when the message was created */
+    val createdAt: Date? = null,
+    /** when the message was created locally */
+    val createdLocallyAt: Date? = null,
+    /** when the message was updated */
+    val updatedAt: Date? = null,
+    /** when the message was updated locally */
+    val updatedLocallyAt: Date? = null,
+    /** when the message was deleted */
+    val deletedAt: Date? = null,
+    /** the users mentioned in this message */
+    val remoteMentionedUserIds: List<String> = emptyList(),
+    /** the users to be mentioned in this message */
+    val mentionedUsersId: List<String> = emptyList(),
+    /** parent id, used for threads */
+    val parentId: String? = null,
+    /** slash command like /giphy etc */
+    val command: String? = null,
+    /** if the message was sent by shadow banned user */
+    val shadowed: Boolean = false,
+    /** if the message is also shown in the channel **/
+    val showInChannel: Boolean = false,
+    // @Embedded(prefix = "channel_info")
+    // val channelInfo: ChannelInfoEntity? = null,
```

Comment:
Why do we comment `channelInfo`? 🤔

Reply:
I left this for later and totally forgot o.O